ppg/pg_tarballs.groovy

```diff
@@ -0,0 +1,291 @@
+
+                        buildStage("oraclelinux:8.9", "--version=${PACKAGE_VERSION} --use_system_ssl=1")
+
+			pushArtifactFolder("tarballs/", AWS_STASH_PATH)
+                    }
+                }
+		stage('Build pg_tarball 15 for OpenSSL 3') {
+                    agent {
+                        label 'docker'
+                    }
+                    steps {
+                        cleanUpWS()
+			def PG_VERSION=15
+                        def BRANCH_NAME = 'REL_15_STABLE'
+			def PACKAGE_VERSION = getPostgreSQLVersion(BRANCH_NAME, "configure.${PG_VERSION}.ssl3")
+
+			def PRODUCT="Percona-PostgreSQL-Tarballs"
+                        def PRODUCT_FULL="${PRODUCT}-${PACKAGE_VERSION}"
+                        AWS_STASH_PATH=UPLOAD/${DESTINATION}/BUILDS/${PRODUCT}/${PRODUCT_FULL}/${BRANCH_NAME}/${TIMESTAMP}
+
+                        buildStage("oraclelinux:8.9", "--version=${PACKAGE_VERSION}")
+
+			pushArtifactFolder("tarballs/", AWS_STASH_PATH)
+                    }
+                }
+		stage('Build pg_tarball 15 for OpenSSL 1.1') {
+                    agent {
+                        label 'docker'
+                    }
+                    steps {
+                        cleanUpWS()
+                        def PG_VERSION=15
+                        def BRANCH_NAME = 'REL_15_STABLE'
+			def PACKAGE_VERSION = getPostgreSQLVersion(BRANCH_NAME, "configure.${PG_VERSION}.ssl1.1")
+
+			def PRODUCT="Percona-PostgreSQL-Tarballs"
+                        def PRODUCT_FULL="${PRODUCT}-${PACKAGE_VERSION}"
+                        AWS_STASH_PATH=UPLOAD/${DESTINATION}/BUILDS/${PRODUCT}/${PRODUCT_FULL}/${BRANCH_NAME}/${TIMESTAMP}
+
+                        buildStage("oraclelinux:8.9", "--version=${PACKAGE_VERSION} --use_system_ssl=1")
+
+			pushArtifactFolder("tarballs/", AWS_STASH_PATH)
+                    }
+                }
+		stage('Build pg_tarball 14 for OpenSSL 3') {
+                    agent {
+                        label 'docker'
+                    }
+                    steps {
+                        cleanUpWS()
+                        def PG_VERSION=14
+                        def BRANCH_NAME = 'REL_14_STABLE'
+			def PACKAGE_VERSION = getPostgreSQLVersion(BRANCH_NAME, "configure.${PG_VERSION}.ssl3")
+
+			def PRODUCT="Percona-PostgreSQL-Tarballs"
+                        def PRODUCT_FULL="${PRODUCT}-${PACKAGE_VERSION}"
+                        AWS_STASH_PATH=UPLOAD/${DESTINATION}/BUILDS/${PRODUCT}/${PRODUCT_FULL}/${BRANCH_NAME}/${TIMESTAMP}
+
+                        buildStage("oraclelinux:8.9", "--version=${PACKAGE_VERSION}")
+
+			pushArtifactFolder("tarballs/", AWS_STASH_PATH)
+                    }
+                }
+		stage('Build pg_tarball 14 for OpenSSL 1.1') {
+                    agent {
+                        label 'docker'
+                    }
+                    steps {
+                        cleanUpWS()
+                        def PG_VERSION=14
```

Comment:
I suggest adding PG_VERSION as a param. So we can pass it from the parameters to the job